ui/README.md

```diff
@@ -0,0 +1,225 @@
+# LiteLLM UI - Frontend & Documentation
+
+This directory contains both the **Dashboard UI** (Next.js) and **Documentation** (Docusaurus) served together via nginx in a Docker container.
+
+## 📁 Project Structure
+
+```
+ui/
+├── litellm-dashboard/     # Next.js dashboard application
+├── docs/                  # Docusaurus documentation
+├── Dockerfile            # Builds both apps and serves with nginx
+├── nginx.conf            # Nginx configuration for routing
+└── README.md             # This file
+```
+
+## 🚀 Quick Start
+
+### Prerequisites
+- Docker installed
+- LiteLLM backend running on port 4000
+
+### 1. Start Backend
+```bash
+# In project root - start the LiteLLM backend
+litellm --config proxy_server_config.yaml
+# Backend will run on http://localhost:4000
+```
+
+### 2. Build and Run Frontend Container
+```bash
+cd ui/
+
+# Build the Docker image
+docker build -t litellm-ui:latest .
+
+# Run the container
+docker run -d --name litellm-ui -p 8080:80 litellm-ui:latest
```

Comment:
Why do we need to map to port 8080? `80:80` should be ok (then u can access the FE with `http://localhost` directly)